src/openshift/storage.ts

```diff
@@ -21,11 +20,14 @@ export class Storage extends OpenShiftItem {
 
         if (!storageName) return null;
 
-        const mountPath = await vscode.window.showInputBox({prompt: "Specify the mount path", validateInput: (value: string) => {
-            if (validator.isEmpty(value.trim())) {
-                return 'Invalid mount path';
+        const mountPath = await vscode.window.showInputBox({
+            prompt: "Specify the mount path",
+            validateInput: (value: string) => {
+                let validationMessage = Storage.emptyName('Empty mount path', value.trim());
+                if (!validationMessage) validationMessage = Storage.isPosixAbsolute('Invalid mount path', value.trim());
+                return validationMessage;
```

Comment:
There should be additional validation to check if provided path uses correct characters. Now it is possible to use any characters even back slash.

Reply:
Also it is not enough to say `Invalid mount path`, because it gives no clue why? It should say literally 'Not absolute mount path'.